tests/test_client_regress/tests.py

```diff
@@ -141,11 +141,15 @@ def test_contains(self):
         try:
             self.assertContains(response, "thrice")
         except AssertionError as e:
-            self.assertIn("Couldn't find 'thrice' in response", str(e))
+            self.assertIn(
+                "False is not true : \"'thrice'\" not found in 'thrice'", str(e)
+            )
         try:
             self.assertContains(response, "thrice", msg_prefix="abc")
         except AssertionError as e:
-            self.assertIn("abc: Couldn't find 'thrice' in response", str(e))
+            self.assertIn(
+                "False is not true : abc: \"'thrice'\" not found in 'thrice'", str(e)
```

Comment:
This doesn't seem right, I would expect to see the response which doesn't contain the word `thrice`. It looks like the wrong thing has been added in here